focus-group.js

```diff
@@ -28,13 +32,25 @@ export function focusGroupKeyUX(options) {
       }
     }
 
+    function getItems(eventTarget, group) {
+      if (group.role === "toolbar") return getToolbarItems(group)
+      return group.querySelectorAll(`[role=${eventTarget.role}]`)
+    }
+
+    function getToolbarItems(group) {
```

Comment:
This function can be moved out of the `focusGroupKeyUX()` body

Reply:
Can you please clarify which functions you would like to move out where `getToolbarItems` or al the "helpers" like `findGroupNodeByEventTarget`, `getItems`, `getToolbarItems`, `isHorizontalOrientation` ?

Reply:
I think I don't get it. 

It seems that all functions (`findGroupNodeByEventTarget`, `getItems`, `getToolbarItems`, `isHorizontalOrientation`) could be moved outside because they are pure and don't rely on local variables from  `focusGroupKeyUX()`. 

Am I right? Some article I could read about it would be appreciated.

Reply:
Hm. Let’s do it separately. I will move after accepting PR.